packages/manager/apps/container/src/container/nav-reshuffle/sidebar/navigation-tree/root.ts

```diff
@@ -1,34 +1,94 @@
-import services from './services';
+import bareMetalCloud from './services/bareMetalCloud';
+import hostedPrivateCloud from './services/hostedPrivateCloud';
+import publicCloud from './services/publicCloud';
+import webCloud from './services/webCloud';
+import network from './services/network';
+import telecom from './services/telecom';
 
 export default {
+  id: 'home',
+  translation: 'sidebar_home',
+  routing: {
+    application: 'hub',
+  },
+  count: false,
   children: [
+    bareMetalCloud,
+    hostedPrivateCloud,
+    publicCloud,
+    webCloud,
+    network,
+    telecom,
     {
-      id: 'home',
-      translation: 'sidebar_home',
+      id: 'sunrise',
+      translation: 'sidebar_sunrise',
       routing: {
-        application: 'hub',
+        application: 'sunrise',
       },
       count: false,
+      feature: 'sunrise',
+      region: ['EU', 'CA'],
```

Comment:
Why not feature flipp this ?

Reply:
we feature-flip it :) Forgot to remove old `region` property.